### ISSUE TITLE
Implementation - Create banner page for login workflow

### ISSUE THREAD

**Issue by phirefly** [18F/dol-whd-14c#415]
Dependent on #81 (Banner UI)

Direct link to prototype page for System Use Banner: https://preview.uxpin.com/dbef3aaee7fc9fac51dc1d087e1f9180f3610ab3#/pages/78406869/simulate/sitemap

For reference, here is the full prototype for the Registration Workflow, beginning with the Flow Chart: https://preview.uxpin.com/dbef3aaee7fc9fac51dc1d087e1f9180f3610ab3#/pages/78406867/simulate/sitemap

[content edited by @simplyshang 11/15]

## Acceptance Criteria

> **NOTE**: Remove any accessibility criteria that don't apply to this user story.  For the full list of accessibility requirements, see [issue #198, Fully Accessible Experience](#198).

#### Functionality

- [ ] Display of new card format
- [ ] Display Read more details content
- [ ] Routing

#### Accessibility
Accessibility per #198

**Comment by phirefly** [comment]
@simplyshang A couple things that will need your input: 
1) Breadcrumbs are currently off -- displaying `Login` instead of generic `14c` landing page
2) We will need some verbiage after `Create Account` and `Login`

**Comment by simplyshang** [comment]
@phirefly updated the main body of the story w/links to the full prototype, and a direct link to the page w/the System Use Banner.

**Comment by EStriegel** [comment]
Complete.  Moving to `Done`
